fix: corrected anchor tags href in Playground sidebar

Santhoshmani1 opened the pull request:
<!--   Thank you for your contribution. Before you submit the pull request:
1. Follow contributing guidelines, the recommended Git workflow, and any related documentation.
2. Test your changes and attach their results to the pull request.
3. Update the relevant documentation.
-->
**Description**

Changes proposed in this pull request:
- Fixed  anchor links in playground Sidebar by adding id to the operations.

**Related issue(s)**

**Fixes**
-  #830 HTML anchor broken on sidebar
-  #822 Anchors for operations are not working as expected

BTOdell commented:
This doesn't appear to fix the problem correctly. You're adding a new anchor instead of fixing the existing anchor with the format: `#operation-{label}-{channel}`.

derberg commented:
@Santhoshmani1 please have a look at comment from @BTOdell

Santhoshmani1 commented:
https://github.com/asyncapi/asyncapi-react/assets/119673958/38569d09-61c7-4042-8e3a-fc7c76cb9318

I have changed the href attribute of sidebar links and added the id attribute to match in the operations. It's working correctly as expected and fixes the problem.

derberg commented on library/src/containers/Sidebar/Sidebar.tsx:
```diff
@@ -360,7 +360,7 @@ const OperationsList: React.FunctionComponent = () => {
   channelName: string;
   summary: string;
-  kind: 'publish' | 'subscribe';
+  kind: 'subscribe' | 'publish';
```
why is this change needed?

derberg commented on library/src/containers/Sidebar/Sidebar.tsx:
```diff
@@ -383,7 +383,7 @@ const OperationItem: React.FunctionComponent<OperationItemProps> = ({
       <a
         className="flex no-underline text-gray-700 mb-2 hover:text-gray-900"
-        href={`#operation-${kind}-${channelName}`}
+        href={'#' + channelName}
```
@AceTheCreator do you recall how will it affect v2 vs v3 compatibility?

@Santhoshmani1 I see in your video that you text with AsyncAPI v2. Can you see how it works with v3 -> https://github.com/asyncapi/spec/blob/master/examples/streetlights-kafka-asyncapi.yml

I'm not sure change is so trivial as in case of v2 operations are part of channels, while in v3 they are not, and there is no pub/sub kinds 🤔 

nevertheless for sure it is not enough just to fix href here. For example when you will check the v3 document that I shared with you, the operation id is `operation-receive-receiveLightMeasurement` where `receive` is `kind`, and `receiveLightMeasurement` is not `channelName` but it is the name of operation. You probably need to extend `OperationItem` component to also have access to operation name, and us it in href 🤔

kennethaasan commented:
@Santhoshmani1, I just wanted to say thank you for the contribution 🚀
